src/core/api/tasklet_api.cpp

```diff
@@ -52,3 +52,10 @@ bool otTaskletsArePending(otInstance *aInstance)
 {
     return aInstance->mIp6.mTaskletScheduler.AreTaskletsPending();
 }
+
+#ifndef _MSC_VER
```

Comment:
@jwhui Can we then accept this fact and use this ifdefs, or do you see some more clever way - like creating a new .cpp file with weak functions that is not built by MSVS?